email/tags.c

```diff
@@ -194,7 +195,7 @@ bool driver_tags_replace(struct TagList *head, char *tags)
     struct ListNode *np = NULL;
     STAILQ_FOREACH(np, &hsplit, entries)
     {
-      driver_tags_add(head, np->data);
+      driver_tags_add(head, mutt_str_dup(np->data));
     }
     mutt_list_free(&hsplit);
```

Comment:
We could cheat a little here:
- drop the `mutt_str_dup()`
- use `mutt_list_clear()` instead
- (and maybe add a comment to explain what's going on :-)